tests/pytests/test_aggregate_params.py

```diff
@@ -85,3 +85,24 @@ def test_apply(env):
         res2 = env.cmd('ft.aggregate', 'idx', '@breed:($p1*|$p2*|$p3*|$p4*)', 'PARAMS', '8', 'p1', 'Dal', 'p2', 'Poo', 'p3', 'Ru', 'p4', 'Mo', 'SORTBY', '1', '@name')
         env.assertEqual(res2, res1)
 
+        # Tag autoescaping
+        if dialect == 5:
+            res1 = env.cmd('ft.aggregate', 'idx', '@code:{ca?33-22}',
+                           'GROUPBY', '1', '@code',
+                           'REDUCE', 'COUNT', 0, 'AS', 'total')
+            env.assertEqual(res1, [1, ['code', 'ca?33-22', 'total', '3']])
+            res2 = env.cmd('ft.aggregate', 'idx', '@code:{$p1}',
+                           'PARAMS', '2', 'p1', 'ca?33-22', 
+                           'GROUPBY', '1', '@code',
+                           'REDUCE', 'COUNT', 0, 'AS', 'total')
+            env.assertEqual(res2, res1)
+
+            res1 = env.cmd('ft.aggregate', 'idx', "@code:{*:99-##}",
+                           'GROUPBY', '1', '@code',
+                           'REDUCE', 'COUNT', 0, 'AS', 'total')
+            env.assertEqual(res1, [1, ['code', 'ca:99-##', 'total', '2']])
+            res2 = env.cmd('ft.aggregate', 'idx', "@code:{*$p1}",
```

Comment:
Can also test a literal star as part of the parameter value (as full match and as prefix/suffix)

Reply:
New tests:
- [test_aggregate_params.py:111](https://github.com/RediSearch/RediSearch/pull/4433/commits/535c73ad18ddf43a9e092a8d280492eab4be4124#diff-4dfcf4e06e9ba073365dc052e7e19bc26a7f6347e3310e636d7cf718d8ae5360R111)